src/components/pages/DAOcreator/index.tsx

```diff
@@ -50,20 +53,22 @@ interface DAO_CREATOR_STATE {
 
 class DAOcreator extends React.Component<Props, State> {
   form = new DAOForm();
+  savedForm = new DAOForm();
```

Comment:
To help with traceability, let's rename this to `recoveredForm` so it matches the other variables that correlate with it?